house/models.py

```diff
@@ -58,9 +57,9 @@ class House(models.Model):
     description = models.TextField(max_length=250, default='')
 
     @staticmethod
-    def create_house(name, public, country, city, parent_profession_1, parent_profession_2, income, children):
+    def create_house(user, public, country, city, parent_profession_1, parent_profession_2, income, children):
```

Comment:
Doesn't house should have a name also?
in addition to a user

Reply:
I don't really see a reason to give the house another name
It's with one to one relation with the User so I think it will be a good fit to use the username here